ow_plexil/src/plexil-adapter/OwInterface.h

```diff
@@ -233,9 +233,11 @@ class OwInterface : public PlexilInterface
     {"HARDWARE_ERROR", std::make_pair(1, false)}
   };
 
-  FaultMap32 m_panTiltErrors = {
-    {"HARDWARE_ERROR", std::make_pair(1, false)},
-    {"JOINT_LIMIT_ERROR", std::make_pair(2, false)}
+  FaultMap64 m_panTiltErrors = {
+    {"PAN_JOINT_LOCKED", std::make_pair(
+      owl_msgs::PanTiltFaultsStatus::PAN_JOINT_LOCKED, false)},
+    {"TILT_JOINT_LOCKED", std::make_pair(
+      owl_msgs::PanTiltFaultsStatus::TILT_JOINT_LOCKED, false)}
```

Comment:
I assume there is a good reason the fault names are changed here. I do not recognize `HARDWARE_ERROR` and `JOINT_LIMIT_ERROR`. Are these left over from an old version of our faults system?

Reply:
Right, the changes are according to the Telemetry Unification Document. The old versions were never properly implemented anyway, but I've implemented these new ones.